WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -313,6 +375,19 @@ private boolean loadMediaId(int mediaId) {
         return true;
     }
 
+    private MediaModel getMediaModelFromEditorImageMetaData() {
+        MediaModel mediaModel = new MediaModel();
+        mediaModel.setUrl(mEditorImageMetaData.getSrc());
+        mediaModel.setTitle(mEditorImageMetaData.getTitle());
+        mediaModel.setCaption(mEditorImageMetaData.getCaption());
+        mediaModel.setAlt(mEditorImageMetaData.getAlt());
+        mediaModel.setFileName(mEditorImageMetaData.getSrc().substring(mEditorImageMetaData.getSrc().lastIndexOf("/") + 1));
```

Comment:
Should we null check `getSrc()` here before calling `substring()` on it?

Reply:
Technically, the `src` should never be null, because we are making json internally ([here])(https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/MetadataUtils.java#L22) and always use empty strings as the default value. But you are right, and it's probably better to be safe, in case it's going to get changed at some point.